openeo/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.32.0a1"
+__version__ = "0.33.0a1"
```

Comment:
bumping the major/minor/patch level of the version is done at time of doing a release, don't change those from a PR.
You can bump the alpha level however

```suggestion
__version__ = "0.32.0a2"
```